client/src/App.js

```diff
@@ -22,7 +23,24 @@ export default class App extends Component {
     return (
       <div>
         <SavedList list={this.state.savedList} />
-        <div>Replace this Div with your Routes</div>
+        <Route exact path="/" 
+        render={props => (
+          <MovieList
+            {...props}
+            // match={props.match}
+            // location={props.location}
+            // history={props.history}
+            items={this.state.savedList}
+          />
+        )} />
+      />
```

Comment:
you've got a loose bit of text here on LINE 36 that is rendering when you select your movie. the whole line should be removed, as the `/>` on LINE 35 is closing your MovieList component